lib/neo4j/tasks/migration.rake

```diff
@@ -21,4 +21,44 @@ namespace :neo4j do
       migration.migrate
     end
   end
+
+  desc 'A shortcut for neo4j::migrate::all'
+  task :migrate do
+    Rake::Task['neo4j::migrate::all'].invoke
```

Comment:
I thought you only did a single `:` between namespaces in rake.  Can you do both?